assets/javascripts/discourse/components/query-result.js.es6

```diff
@@ -187,12 +188,7 @@ const QueryResultComponent = Ember.Component.extend({
 
   @computed("content.rows.[]")
   graphValues(rows) {
-    return rows.map((r) => r[1]);
-  },
-
-  @computed("columnDispNames.[]")
-  graphDatasetName(columnDispNames) {
-    return columnDispNames[1];
+    return rows.mapBy(1);
```

Comment:
there's a mapBy computed property in ember https://api.emberjs.com/ember/release/functions/@ember%2Fobject%2Fcomputed/mapBy